src/middlewared/middlewared/alert/source/nfs_bindaddr.py

```diff
@@ -7,6 +7,7 @@ class NFSBindAlertClass(AlertClass):
     title = "NFS Services Could Not Bind to Specific IP Addresses, Using 0.0.0.0"
     text = "NFS services could not bind to specific IP addresses, using 0.0.0.0."
 
+    run_on_backup_node = False
```

Comment:
I think this should be in AlertSource

Reply:
@william-gr sorry, thank you!

Reply:
I meant this attribute should be in t he class below for it to work

Reply:
damnit, fixed